source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -823,6 +843,18 @@ Finish installation
 
 #. When the system reboots, remove any installation media present.
 
+#. Upon launch of the GUI, open a Terminal.
+
+#. In the Terminal, log into `root` for the first time.
+
+   .. code-block:: bash
+
+      sudo -i
```

Comment:
@mvincerx I couldn't verify this command on my system to see if it prompts me to change the password.  The rest of the document looks good though.

Reply:
@tpleavitt. Please see my latest push that includes revisions to `enable-user-space`.

Reply:
@mvincerx , I just did a clean desktop-install and brought up a command window and type "sudo -i", this took me to the root user prompt "root@clr-toml-test # ", so I'm root, it didn't prompt to add a root password, so this method appears to not be working.  If I enter the command "passwd" it will prompt me to enter the new password and verify it

Reply:
So I tested what happens if:
-	No admin user is assigned
The result is that ONLY the [long hash +] `login:` appears
There isn’t any guidance. 
Therefore, I was right to add the note under “Recommended options’.  And like we discussed, I fleshed out the language more with`...full desktop experience` in intro.

Reply:
Another nice feature. 
Suppose I continue as above. Now, I only run:
`swupd bundle-add `desktop-autostart``

Next I reboot. 

This time after rebooting, a new dialogue box appears--automically--that requests that I create a new user.  (That's new along with the TUI.)

Reply:
Revised per feedback from @odwdinc. Thanks again for doing this.

Reply:
Can I get a witness?